be/src/runtime/fragment_mgr.cpp

```diff
@@ -455,18 +455,6 @@ void FragmentMgr::remove_pipeline_context(
     std::lock_guard<std::mutex> lock(_lock);
     auto query_id = f_context->get_query_id();
     auto* q_context = f_context->get_query_context();
-    bool all_done = q_context->countdown();
```

Comment:
You can remove `countdown()` from `QueryContext`

Reply:
done